src/ClassSet.php

```diff
@@ -7,15 +7,15 @@
 
 class ClassSet implements \IteratorAggregate
 {
-    /** @var string */
-    private $directory;
+    /** @var string[] */
+    private $directoryList;
```

Comment:
I was thinking of a var name like `directories` instead of `directoryList`, what do you think?

Reply:
It does read better, however when plural prefer to suffix with "List", otherwise in most cases the difference between plural and singular will be only a letter (s), wich can easily be overlooked and result in confusion.

That said, if my argument doesn't convince you let me know and I will gladly change it. :)

And tkx for the CR btw.